### PR TITLE
Fix some typos in README

### PR DESCRIPTION
**Pull request by HenryRLee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## What
 
 `lldb-eval` is an [LLDB](https://lldb.llvm.org/)-based library for evaluating
-debug expressions in the context of the debuggee process. All modern debuggers
+debug expressions in the context of the debugger process. All modern debuggers
```
Comment (werat) [inline README.md:8]: "debuggee" is correct here, the expressions are evaluated in the context of the "process that is being debugged". Let's just use "debugged", if debuggee looks confusing :)

```suggestion
debug expressions in the context of the debugged process. All modern debuggers
```

**Review by werat — CHANGES_REQUESTED** [review]
Thank you for your contribution!

**Comment by werat** [comment]
disregard the failed benchmark action, it's not working for new contributors unfortunately

**Comment by HenryRLee** [comment]
Thanks for the feedback @werat. I updated the PR accordingly.

**Comment by werat** [comment]
Thanks again for the contribution!

**Pull request merged**